<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="searchGroupBox">
       <property name="title">
        <string>Searching and Font Customization</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="searchEdit">
            <property name="styleSheet">
             <string notr="true">font: italic 8pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Type here to search...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="indexListWidget"/>
          </item>
          <item>
           <widget class="QPushButton" name="fontButton">
            <property name="text">
             <string>Change Font and Font Size</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeWidget" name="indexTreeWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <column>
           <property name="text">
            <string>Table of Contents</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Digital Systems and Binary Numbers</string>
           </property>
           <item>
            <property name="text">
             <string>Digital Systems</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Binary Numbers</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Number-Base Conversion</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Complement Of Number</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Signed Binary Number</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Binary Codes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Binary Logic</string>
            </property>
           </item>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="moreButton">
          <property name="text">
           <string>&amp;&lt;&lt; More</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWebView" name="webView">
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport_to_PDF"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuAbout_ATeL">
     <property name="title">
      <string>About</string>
     </property>
     <addaction name="actionATeL"/>
     <addaction name="actionDeveloper"/>
     <addaction name="actionQt"/>
    </widget>
    <addaction name="actionUser_Guide"/>
    <addaction name="separator"/>
    <addaction name="menuAbout_ATeL"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionNumber_Base_Converter"/>
    <addaction name="actionNumber_Base_Chart"/>
    <addaction name="separator"/>
    <addaction name="actionK_Map_Generator_Logic_Minimizer_Soon"/>
    <addaction name="actionLogic_Tester_Soon"/>
    <addaction name="actionLearning_Tracker"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExport_to_PDF">
   <property name="text">
    <string>Export to PDF (Soon..)</string>
   </property>
  </action>
  <action name="actionUser_Guide">
   <property name="text">
    <string>User Guide</string>
   </property>
  </action>
  <action name="actionNumber_Base_Converter">
   <property name="icon">
    <iconset resource="appresource.qrc">
     <normaloff>:/images/qbase_operation.svg</normaloff>:/images/qbase_operation.svg</iconset>
   </property>
   <property name="text">
    <string>Number Base Converter</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_Developer">
   <property name="text">
    <string>About Developer</string>
   </property>
  </action>
  <action name="actionATeL">
   <property name="text">
    <string>ATeL</string>
   </property>
  </action>
  <action name="actionDeveloper">
   <property name="text">
    <string>Developer</string>
   </property>
  </action>
  <action name="actionQt">
   <property name="text">
    <string>Qt</string>
   </property>
  </action>
  <action name="actionK_Map_Generator_Logic_Minimizer_Soon">
   <property name="text">
    <string>K-Map Generator and Logic Minimizer (Soon...)</string>
   </property>
  </action>
  <action name="actionLogic_Tester_Soon">
   <property name="text">
    <string>Logic Tester (Soon...)</string>
   </property>
  </action>
  <action name="actionLearning_Tracker">
   <property name="text">
    <string>Learning Tracker (Soon...)</string>
   </property>
  </action>
  <action name="actionNumber_Base_Chart">
   <property name="icon">
    <iconset resource="appresource.qrc">
     <normaloff>:/images/charticon.svg</normaloff>:/images/charticon.svg</iconset>
   </property>
   <property name="text">
    <string>Number Base Chart</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="appresource.qrc"/>
 </resources>
 <connections/>
</ui>
